debug.log

```diff
@@ -0,0 +1 @@
+[0202/145128.544:ERROR:crash_report_database_win.cc(568)] CreateDirectory C:\Users\JOE~1.SKE\AppData\Local\Temp\karma-96591541\Crashpad: The system cannot find the path specified. (0x3)
```

Comment:
Not sure if we want to include this log file.

Reply:
yeah, we should delete this for sure

Reply:
done